system-test/test-e2e.ts

```diff
@@ -408,6 +412,7 @@ describe('@google-cloud/debug end-to-end behavior', () => {
 
           console.log('-- waiting before checking if the log was written');
           return Promise.all([breakpoint, delay(10 * 1000)]);
+          // TODO: Determine if the results parameter should be used.
```

Comment:
What is this TODO in reference to?

Reply:
I copied this in order to make lines 338 to 417 identical to lines 146 to 225. I guess it's in reference to whether we should remove `results`, since nothing uses it.